### ISSUE TITLE
Missing libigl cmake finder in data_preprocessing

### ISSUE THREAD

**Issue by isaak-l** [Colin97/DeepMetaHandles#1]
Thank you for providing the code to your paper.

I noticed that a note in the README is missing that tells the user to clone libigl into `data_preprocessing` (as in the libigl example project). Furthermore, the [cmake finder](https://github.com/libigl/libigl-example-project/blob/master/cmake/FindLIBIGL.cmake) is missing in `data_preprocessing`.

This would make it even easier for other researchers to use your code on other datasets.

**Comment by averysi224** [comment]
Hi,

I'm not familiar with cpp. I've cloned libigl in the data_processing folder and as well as cmake it. However when compile data preprocessing I encounter this:
CMake Error at CMakeLists.txt:15 (add_executable):
  Target "calc_weight_bin" links to target "igl::opengl" but the target was
  not found.  Perhaps a find_package() call is missing for an IMPORTED
  target, or an ALIAS target is missing?


CMake Error at CMakeLists.txt:15 (add_executable):
  Target "calc_weight_bin" links to target "igl::opengl_glfw" but the target
  was not found.  Perhaps a find_package() call is missing for an IMPORTED
  target, or an ALIAS target is missing?
May I ask for some suggestions? Thanks!

**Comment by Colin97** [comment]
Did you "make" & "make install" libigl?

> Hi,
> 
> I'm not familiar with cpp. I've cloned libigl in the data_processing folder and as well as cmake it. However when compile data preprocessing I encounter this:
> CMake Error at CMakeLists.txt:15 (add_executable):
> Target "calc_weight_bin" links to target "igl::opengl" but the target was
> not found. Perhaps a find_package() call is missing for an IMPORTED
> target, or an ALIAS target is missing?
> 
> CMake Error at CMakeLists.txt:15 (add_executable):
> Target "calc_weight_bin" links to target "igl::opengl_glfw" but the target
> was not found. Perhaps a find_package() call is missing for an IMPORTED
> target, or an ALIAS target is missing?
> May I ask for some suggestions? Thanks!

**Comment by averysi224** [comment]
Yes. Could you please provide all the command lines for this step? I'll really appreciate it.

**Comment by Silverster98** [comment]
> Yes. Could you please provide all the command lines for this step? I'll really appreciate it.

I have tried the following steps, and it works well.
1. clone [libigl](https://github.com/libigl/libigl) to `data_preprocessing/`
2. copy the content of [cmake finder](https://github.com/libigl/libigl-example-project/blob/main/cmake/libigl.cmake) to `./data_preprocessing/cmake/FindLIBIGL.cmake`
3. execute `cmake .`
4. execute `make`
